### DISCUSSION TITLE
posts/dummy/

### DISCUSSION THREAD

**Discussion by giscus** [rasnes/homepage-comments#2] [Announcements]
# posts/dummy/

Welcome to my blog!

https://rasmusnes.com/posts/dummy/

**Comment by rasnes** [comment]
Testing the comment field. How will this look?

**Reply by rasnes** [reply]
I looks good. Perhaps except from the minor aesthetic thing, that the dark mode of Github is darker than my theme.

**Comment by rasnes** [comment]
But, there is one thing, it has too much whitespace between the footer and comment section. That should be decreased.

**Comment by rasnes** [comment]
Posting from mobile. Works okay I think.
